src/index.ts

```diff
@@ -1,2 +1,12 @@
-export * from './dsl/index';
-export * from './model/index';
+import * as dsl from './dsl/index';
+import * as model from './model/index';
+
+const oas30 = { ...model.oas30, ...dsl.oas30 };
+const oas31 = { ...model.oas31, ...dsl.oas31 };
```

Comment:
I believe these two guys should also be exported (not within the "default"), then they will be importable individually, @pjmolina 

```suggestion
export const oas30 = { ...model.oas30, ...dsl.oas30 };
export const oas31 = { ...model.oas31, ...dsl.oas31 };
```

Reply:
I'm going to try it locally and then I'll confirm.

Reply:
Yes, I confirm, with this cache individual imports do work

![image](https://user-images.githubusercontent.com/13189514/228617085-dcd390b7-ce7e-46d8-9682-a10f19a806c1.png)

Reply:
However, this does not solve the import problem for types (in model)

Reply:
perhaps these entities (`oas30` and `oas31`) should be namespaces, not constants.
Thus, they could also contain types ("model")

![image](https://user-images.githubusercontent.com/13189514/228618846-c4495ea7-50e6-408f-bed1-b58f91e4a14b.png)

Reply:
I'm going to take a bit time to prepare a better suggestion